source/documentors/references/templates/doc_concept_template.rst

```diff
@@ -20,14 +20,14 @@ Since this type of document seeks to explain a topic, you will likely need to ad
 External Links
 --------------
 
-You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
+You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
```

Comment:
Can we make this an unchanging link?

Reply:
This is an example link for how to do inline links and indirect links.